.golangci.yml

```
@@ -70,3 +69,20 @@ issues:
     - linters:
         - wsl
       text: "only one cuddle assignment allowed before range statement"
+
+    # Ignore new line warning at L30. New line is added to reduce line length according to lll
+    - linters:
+        - stylecheck
+      text: "ST1005: "
+      path: pkg/networkplugin-syncer/handlers/ovn/handler.go
+
+    # Allow dot imports for `fake` directory
+    - linters:
+        - stylecheck
+      text: "ST1001: "
+      path: pkg/cable/fake
+
+    - linters:
+        - stylecheck
+      text: "ST1001: "
+      path: pkg/cableengine/fake
```

We can handle all _fake_ dirs with one rule. Also need to exclude _test/e2e_.

```suggestion
    # Allow dot imports for gingko and gomega in test-related directories
    - linters:
        - stylecheck
      text: "ST1001: "
      path: /fake

    - linters:
        - stylecheck
      text: "ST1001: "
      path: test/e2e
```